Source/Core/Core/PowerPC/PPCSymbolDB.cpp

```diff
@@ -44,8 +44,8 @@ Common::Symbol* PPCSymbolDB::AddFunction(const Core::CPUThreadGuard& guard, u32
   if (!PPCAnalyst::AnalyzeFunction(guard, start_addr, symbol))
     return nullptr;
 
-  m_functions[start_addr] = std::move(symbol);
-  Common::Symbol* ptr = &m_functions[start_addr];
+  const auto insert = m_functions.emplace(start_addr, std::move(symbol));
+  Common::Symbol* ptr = &insert.first->second;
```

Comment:
I suppose there is no advantage to use structured bindings with `emplace` instead? (since we'll have to ignore the `bool` result which isn't required when using `.first` from the `tuple` result directly)

Feel free to ignore this comment.

Reply:
yeah, mainly because ignoring the bool result can cause warnings